src/nrfx/drivers/nrfx_common.h

```diff
@@ -50,6 +50,16 @@
 extern "C" {
 #endif
 
+/*
+ * Suppress use of anomaly workarounds in nrfx drivers that would directly
+ * access hardware registers.
+ */
+#define USE_WORKAROUND_FOR_ANOMALY_132 0
+
+#ifndef __STATIC_INLINE
```

Comment:
Where are you getting this one from in the real nRFx? from CMSIS?

Reply:
Yes, from CMSIS.